src/index.js

```diff
@@ -23,6 +23,11 @@ export async function run() {
       await execShellCommand('brew install tmate');
     } else if (process.platform === "win32") {
       await execShellCommand('pacman -Sy --noconfirm tmate');
+
+      const bashProfilePath = "C:\\msys64\\etc\\profile"
+      let bashProfileContent = fs.readFileSync(bashProfilePath).toString()
+      bashProfileContent += '\ncd "${GITHUB_WORKSPACE}"\n'
+      fs.writeFileSync(bashProfilePath, bashProfileContent)
```

Comment:
That's an ugly workaround. Does `CHERE_INVOKING` no longer work?

Reply:
Figured it out: `tmate` is called within the login shell that receives _and unsets_ `CHERE_INVOKING`, and `tmate` _then_ calls _another_ login shell (that no longer sees `CHERE_INVOKING`, of course).

Reply:
makes sense! then we don't need this ugly workaround :D